src/generator/AutoRest.Ruby.Azure.Tests/RspecTests/paging_spec.rb

```diff
@@ -129,6 +129,21 @@
     expect(items.count).to eq(10)
   end
 
+  it 'should get multiple pages with fragmented next link' do

```

Comment:
Added a test in AutoRest to cover paging method with operation name and grouped parameters. 

@tbombach Could you also please review the changes in second commit to make sure i did things right? 

Thanks